services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -37,15 +53,35 @@
 async def list_jobs(
     solver_key: SolverKeyId,
     version: str,
-    user_id: int = Depends(get_current_user_id),
+    user_id: PositiveInt = Depends(get_current_user_id),
     catalog_client: CatalogApi = Depends(get_api_client(CatalogApi)),
+    webserver_api: AuthSession = Depends(get_webserver_session),
     url_for: Callable = Depends(get_reverse_url_mapper),
 ):
     """ List of all jobs in a specific released solver """
-    from .jobs_faker import list_jobs_impl
 
-    solver = await catalog_client.get_solver(user_id, solver_key, version)
-    return await list_jobs_impl(solver.id, solver.version, url_for)
+    async def _fake_impl():
+        from ._jobs_faker import list_jobs_impl
+
+        solver = await catalog_client.get_solver(user_id, solver_key, version)
+        return await list_jobs_impl(solver.id, solver.version, url_for)
```

Comment:
Am I correct to assume this is no longer used here?
The same goes for all these functions inside this file.

Reply:
yes. I just keep it for the moment to compare tests. Probably I will recycle everything in subsequent PRs to produce API mock fixtures for testing

Reply:
ok.. had the same question...